gwcs/wcs.py

```diff
@@ -27,7 +27,51 @@
     HAS_FIX_INPUTS = False
 
 
-__all__ = ['WCS']
+__all__ = ['WCS', 'NoConvergence']
+
+class NoConvergence(Exception):
+    """
+    An error class used to report non-convergence and/or divergence
+    of numerical methods. It is used to report errors in the
+    iterative solution used by
+    the :py:meth:`~astropy.wcs.WCS.all_world2pix`.
+
+    Attributes
+    ----------
+
+    best_solution : `numpy.ndarray`
+        Best solution achieved by the numerical method.
+
+    accuracy : `numpy.ndarray`
```

Comment:
RMS? No!

It is in the same sense in which an iterative process uses terms "tolerance" or "accuracy" to detect when it can stop iterating. It is "estimated accuracy" of the solution. In this particular case, achieved accuracy is estimated as `|x_n - x_{n-1}|`.

Reply:
I mean that it should be clearly defined in the docstring what accuracy means. Many may interpret it as RMS.